p4-16/spec/P4-16-spec.mdk

```diff
@@ -6568,7 +6568,7 @@ table, just a default action---i.e., the associated lookup table is
 always the empty map.
 
 Each key element can have an optional `@name` annotation which is
-used to synthesize the control-plane visible name for the key field.
+used to synthesize the controlplane-visible name for the key field.
```

Comment:
This looks like a typo.

Reply:
Actually, not a typo but rather “thinking in German”...  I didn｀t think “{control-plane}-visible” would go over well, and I didn｀t want to change the word order, so I just made up a new compound word.  _Warum nicht?_ 😉

Reply:
https://github.com/p4lang/p4-spec/pull/1117/commits/1044acdf12a5f555a9ee742421a70bbc72686744